resources/sass/aante.scss

```diff
@@ -0,0 +1,169 @@
+// $author Ante Aljinovic https://github.com/aljinovic
+
+@import "base";
+
+:root {
+  --main-background: #f8f8f8;
+  --secondary-background: #f8f8f8;
+
+  --variable-type-color: #06f;
+  --variable-type-color-hover: #f00;
+
+  --border-color: #d7d7d7;
+  --border-color-hover: #aaa;
+
+  --alternative-background: #fff;
+  --highlight-color: #cfc;
+
+  --caret-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 150'><path d='M6 7h18l-9 15zm0 30h18l-9 15zm0 45h18l-9-15zm0 30h18l-9-15zm0 12l18 18m-18 0l18-18' fill='%23555'/><path d='M6 126l18 18m-18 0l18-18' stroke-width='2' stroke='%23555'/></svg>");
+
+  @media (prefers-color-scheme: dark) {
+    --main-background: #070707;
+    --secondary-background: #070707;
+
+    --variable-type-color: #ff9900;
+    --variable-type-color-hover: #00ffff;
+
+    --border-color: #282828;
+    --border-color-hover: #555555;
+
+    --backdrop-color: rgba(0, 0, 0, 0.9);
+    --text-color: #e2e1e1;
+    --variable-name-color: #e2e1e1;
+
+    --alternative-background: #000;
+    --highlight-color: #330033;
+
+    --caret-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 150'><path d='M6 7h18l-9 15zm0 30h18l-9 15zm0 45h18l-9-15zm0 30h18l-9-15zm0 12l18 18m-18 0l18-18' fill='%23AAA'/><path d='M6 126l18 18m-18 0l18-18' stroke-width='2' stroke='%23AAA'/></svg>");
+  }
+}
+
+.kint-rich {
+  .kint-focused {
+    box-shadow: 0 0 3px 2px var(--variable-type-color-hover);
+  }
+
+  dt {
+    font-weight: normal;
+
+    &.kint-parent {
+      margin-top: 4px;
+    }
+  }
+
+  dl dl {
+    margin-top: 4px;
+    padding-left: 25px;
+    border-left: none;
+  }
+
+  > dl > dt {
+    background: var(--secondary-background);
+  }
+
+  //
+  // TABS
+  // --------------------------------------------------
+
+  ul {
+    margin: 0;
+    padding-left: 0;
+
+    &:not(.kint-tabs) > li {
+      border-left: 0;
+    }
+
+    &.kint-tabs {
+      background: var(--secondary-background);
+      border: var(--border);
+      border-width: 0 1px 1px 1px;
+      padding: 4px 0 0 12px;
+      margin-left: -1px;
+      margin-top: -1px;
+
+      li,
+      li + li {
+        margin: 0 0 0 4px;
+      }
+
+      li {
+        border-bottom-width: 0;
+        height: calc(var(--spacing) * 6);
```

Comment:
The original was `spacing * 6px + 1px` - there's a gap in the border (On firefox at least) without the extra pixel

Reply:
Man, I am sorry I don't know why I missed these things, very important catch, thanks!

Fixed in the newer commit which I will push soon.